lib/misc.py

```diff
@@ -183,3 +183,24 @@ def undelete(cli):
    if (git_hash is None):
       ch.FATAL("image not in cache")
    bu.cache.checkout(img, git_hash, None)
+
+def upload_cache(cli):
+   #TODO: Make upload-cache an independent file and call functions from there
```

Comment:
May as well go ahead and do that.

Reply:
This should be implemented and working now, however its in a pretty simply made file and I'm sure there is improvement on how to set up a file like this that could be made.